### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by SSEKPIUS**
### Use jest framework for testing.

- [ ] Create a test file(s) ([..].test.js) for a file(s) containing the following functions:

- [ ] a function for editing the task description.

- [ ] a function for updating an item's 'completed' status.

- [ ] the "Clear all completed" function.

Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
Mock the HTML elements to test DOM manipulation functions.
Make sure you group your tests using the describe() method.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @SSEKPIUS and @BryteLitty, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project to which the third coding partner did not contribute.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BryteLitty** [comment]
Hello @julie-ify,

Thank you for your feedback. In response to your comment, our third coding partner has not been available throughout the week that's why he didn't contribute to the project. Please, what do you suggest, now that we know that he's not available?

**Comment by julie-ify** [comment]
@BryteLitty You should contact the Student Success team to resolve this. Perhaps the missing students would have been removed from the group and worked on these projects later. Submitting the project with a missing partner gives the impression that the partner has completed the project which is not the case.

TL;DR - Consider contacting Student Success for the way forward in this case.

**Comment by julie-ify** [comment]
Hi @SSEKPIUS and @BryteLitty,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Consider contacting Student Success for the way forward in this case of a missing coding partner

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by jasonfelice — APPROVED** [review]
Hi @SSEKPIUS, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️
All tests are passing✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
